### PR TITLE
Minimap update

### PR DESCRIPTION
**Pull request by RealBadAngel**
Add setting for scan height, fix object names, add conf examples.

### CODE REVIEW

File: src/minimap.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ void MinimapUpdateThread::Stop()
 
 void MinimapUpdateThread::enqueue_Block(v3s16 pos, MinimapMapblock *data)
 {
-	if (m_queue.addBlock(pos, data))
```
Comment (est31) [inline src/minimap.cpp]: I've made the queue private in 420125debd3d010fcd3e7738c43f25c8f407ab63 . Therefore, the renaming isn't needed. Which sense does it make to make it public, if its not used publicly?

**Comment by RealBadAngel** [comment]
@est31 good point. ive also moved flushing m_blocks_cache to proper place, so it can remain private.

File: src/minimap.cpp [hunk 2]
```diff
@@ -164,6 +164,15 @@ void *MinimapUpdateThread::Thread()
 	return NULL;
 }
 
+MinimapUpdateThread::~MinimapUpdateThread()
+{
+	for (std::map<v3s16, MinimapMapblock *>::iterator
+			it = m_blocks_cache.begin();
+			it != m_blocks_cache.end(); it++){
```
Comment (est31) [inline src/minimap.cpp]: space

**Comment by est31** [comment]
looks good :+1:

**Comment by RealBadAngel** [comment]
https://github.com/minetest/minetest/commit/b160f8dfe7e3bf8176e3f9f64dc2d9a9868d7fce

**Pull request closed without merging**